pennylane/ops/arithmetic/sum.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2018-2022 Xanadu Quantum Technologies Inc.
```

Comment:
Is this file a merge gone wrong?

Reply:
I was doing some funky stuff locally, I had branched: 

`master --> Sum --> SProd --> Prod`

The idea was that I could develop Sum and then pull the changes into SProd so I can test them both together and then pull Sprod into Prod. But it got messy when I was focusing on Sum and we were renaming things.